README.md

```diff
@@ -29,8 +29,7 @@ use std::path::Path;
 
 fn main() {
     let path = Path::new("src").join("target.l");
-    let path = path.to_str().unwrap().to_string();
-    if let Err(e) = rflex::process(path) {
+    if let Err(e) = rflex::process(path, None) {
```

Comment:
Should we show `OUT_DIR` environment variable in README.md such as examples, or not?

Reply:
I forgot it. I think `OUT_DIR` is required because this is an example of `build.rs`.

Reply:
:+1: